### PR TITLE
Create login card and github link card

### LINKED ISSUE #579: [Onboarding Flow]- Implement Before and After GitHub Sign-In Page
# Refactor- Implement Before and After GitHub Sign-In Page in Onboarding flow
## Overview:
The goal of this issue is to enhance the user experience after successful sign-in with GitHub by creating a "Thank You" page. This page will serve to confirm the successful connection of the user's GitHub account and provide information about the next steps they can take on the platform. 

## User Stories:
- As a user who has successfully signed in with GitHub, I expect to be redirected to a "Thank You" page that acknowledges my GitHub account connection.
- As a user, I want the "Thank You" page design to be visually pleasing and consistent.
- As a user, I want guidance on the next steps I should take after connecting my GitHub account.
- As a user, I want to see this page in 1st step of onboarding flow

## Requirement:

**New Implementation:**
Refactor Design:- https://www.figma.com/file/nUbzEXoWgJgUQVlQdiCiog/Status-site?type=design&node-id=45-2410&mode=design&t=93xUWTOzJMLsh2nK-0
****(Note:- information written in this will be change please confirm with other developer)**

------------------------------------------------

**Previous Implementation:**
image

Previous Implementation PR: https://github.com/Real-Dev-Squad/website-my/pull/268

-------------------------------------------------

## Acceptance Criteria:
- [ ] TDD
- [ ] The "Thank You" page is displayed immediately after a successful GitHub sign-in.
- [ ] Everything is inside feature flag

### PR DESCRIPTION
**Pull request by vinit717**
### Issue

closes #579 

### What is the change?

Add a before logged in and after logged in page 
This feature is under feature flag

### Is Development Tested?

- [x] Yes
- [ ] No

### Video 

https://github.com/Real-Dev-Squad/website-www/assets/111434418/b6f62f0d-282c-45fd-b7f2-79233b3687a3

### CODE REVIEW

File: app/components/stepper-signup.js [hunk 1]
```diff
@@ -19,4 +24,18 @@ export default class StepperSignupComponent extends Component {
   @action handleInputChange(key, value) {
     set(this.signupDetails, key, value);
   }
+
+  @action incrementStep() {
+    if (this.currentStep < 5) {
+      this.currentStep += 1;
+      const queryParams = { dev: true, step: 1 };
+      localStorage.setItem('currentStep', this.currentStep);
```
Comment (shubhamsigdar1) [inline app/components/stepper-signup.js:32]: as we discuss we donot store currentstep on localstorage in signup form

there is other way you can do this:- refer (https://github.com/Real-Dev-Squad/website-my/blob/develop/app/templates/new-signup.hbs#L3, https://github.com/Real-Dev-Squad/website-my/blob/develop/app/controllers/new-signup.js#L40)

File: app/components/stepper-signup.js [hunk 2]
```diff
@@ -19,4 +24,18 @@ export default class StepperSignupComponent extends Component {
   @action handleInputChange(key, value) {
     set(this.signupDetails, key, value);
   }
+
+  @action incrementStep() {
+    if (this.currentStep < 5) {
+      this.currentStep += 1;
+      const queryParams = { dev: true, step: 1 };
```
Comment (shubhamsigdar1) [inline app/components/stepper-signup.js:31]: Can you give me brief idea what you trying to do from line 29 to 31

File: app/components/signup-steps/step-zero.hbs [hunk 3]
```diff
@@ -1,26 +1,28 @@
...
+    @onClick={{@letsGoHandler}}
+    @test='lets-go'
+    @type='button'
+  />
+  {{else if  (not this.login.isLoggedIn)}}
+  <p class="signup-page-heading" data-test-signup-heading>You are not logged in. Please log in to continue...</p>
+  <Reusables::Button
+    @text='login'
+    @variant='dark'
+    @onClick={{this.loginwithgithub}}
```
Comment (satyam73) [inline app/components/signup-steps/step-zero.hbs:23]:
```suggestion
    @onClick={{this.loginWithGithub}}

```

File: app/components/signup-steps/step-zero.hbs [hunk 4]
```diff
@@ -1,26 +1,28 @@
...
+  <p class="signup-page-heading" data-test-signup-heading>Thank you for connecting your GitHub</p>
+  <p class="p2-description" data-test-signup-paragraph>Please continue with Sign Up with in order to use features display yourself on members page etc</p> 
+  <Reusables::Button
+    @text="Let's Go"
+    @variant='dark'
+    @onClick={{@letsGoHandler}}
+    @test='lets-go'
+    @type='button'
+  />
+  {{else if  (not this.login.isLoggedIn)}}
```
Comment (satyam73) [inline app/components/signup-steps/step-zero.hbs:18]: else will also work here

File: app/components/signup-steps/step-zero.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+export default class SignupStepsStepZeroComponent extends Component {
+  @service login;
+  @service router;
+  @service fastboot;
+  @service store;
+  @service controller;
+  @tracked currentStep = 0;
+
+  @action loginwithgithub() {
```
Comment (satyam73) [inline app/components/signup-steps/step-zero.js:15]:
```suggestion
  @action loginWithGithub() {
```

File: app/styles/onboarding-card.module.css [hunk 6]
```diff
@@ -1,96 +1,86 @@
...
     flex-direction: column;
     justify-content: center;
     align-items: center;
-    background-color: #FFF;
+    background-color: var(--color-white);
     border-radius: 24px;
     position: relative;
     padding: 0 1rem;
+    margin-top: 40px;
     box-shadow: 0px 4px 10px -2px rgba(16, 24, 40, 0.06), 0px 6px 15px -2px rgba(16, 24, 40, 0.10);
```
Comment (satyam73) [inline app/styles/onboarding-card.module.css:13]: Can we use color variables here?

**Review by satyam73 — APPROVED** [review]
LGTM

**Review by rohan09-raj — APPROVED** [review]

**Pull request merged**